IDE/WIN10/wolfssl-fips.vcxproj

```diff
@@ -267,6 +267,9 @@
     <ClCompile Include="..\..\wolfcrypt\src\memory.c" />
     <ClCompile Include="..\..\src\ocsp.c" />
     <ClCompile Include="..\..\wolfcrypt\src\sha3.c" />
+    <ClCompile Include="..\..\wolfcrypt\src\sp_c32.c" />
+    <ClCompile Include="..\..\wolfcrypt\src\sp_c64.c" />
```

Comment:
sp_c64.c won't work as it needs a 128-bit type.
sp_x86_64.c and sp_x86_asm.asm could be added.

Reply:
This change has been made and branch/PR update.